src/backend/feed/processor.js

```diff
@@ -54,6 +54,7 @@ async function getFeedInfo(feed) {
     lastModified: null,
     contentType: null,
     shouldDownload: true,
+    user: null,
```

Comment:
We need to think about this one.  Probably having it be `null` is right, but it at least needs a comment above to indicate that we don't have user info at this stage (i.e., wiki feeds will be owned by no user), whereas feeds that get added via the REST API will include it via passport.js user detail on the session.

Reply:
So, I was thinking we would be moving the wiki feeds under an admin account or am I wrong?

Reply:
`null` for the user would mean "owned by the admins", yes.

Reply:
Ah - genius.

Reply:
Fixed